webapp/navigation.py

```diff
@@ -25,3 +33,14 @@ def get_current_page_bubble(path):
                     page["active"] = True
 
     return {"page_bubble": current_page_bubble}
+
+
+def build_navigation(id, title):
+    meganav_section = meganav_data[id]
+    html_string = render_template_string(
+        '{% include "navigation/_dropdown.jinja" %}',
```

Comment:
```suggestion
        '{% include "navigation/_dropdown.html" %}',
```
Should it be the html file instead of Jinja? Since the jinja file has been removed in this PR